election/states.go

```diff
@@ -55,6 +55,11 @@ var usc = map[string]string{
 	"WY": "Wyoming",
 }
 
+func HasState(code string) bool {
```

Comment:
Should be renamed to StateExists, otherwise this implies multiple objects containing sets of states.